lib/roast/workflow/workflow_executor.rb

```diff
@@ -18,17 +18,18 @@ def initialize(workflow, config_hash, context_path)
         @context_path = context_path
       end
 
-      def execute_steps(steps)
-        steps.each do |step|
-          case step
+      def execute_steps(workflow_steps)
+        workflow_steps.each do |workflow_step|
+          case workflow_step
           when Hash
-            execute_hash_step(step)
+            execute_hash_step(workflow_step)
           when Array
-            execute_parallel_steps(step)
+            execute_parallel_steps(workflow_step)
           when String
-            execute_string_step(step)
+            execute_string_step(workflow_step)
+            Kernel.binding.irb if workflow.pause_step_name == step
```

Comment:
Hey @nfgrep this used to be extracted to a separate method but....then the context wasn't at this code location and the `step` local wasn't in scope. Move this back and then was able to wrangle Minitest as seen in that file below.